devops/wsi_deid/girder.local.conf

```diff
@@ -59,3 +60,24 @@ folder_name_field = "TokenID"
 image_name_field = ""
 validate_image_id_field = False
 new_token_pattern = "####@@######"
+reject_reasons = [{
```

Comment:
We should probably add a default to this value in the code (so if it isn't set in the config AND `require_reject_reason = True`, you can still redact the image.  Either that or if it isn't set, ignore the value of `require_reject_reason`.

Reply:
Added defaults based on the issue in bd5d4f9